onepassword_secrets.py

```diff
@@ -443,7 +443,7 @@ def push_local_secrets():
 
     env_file_name = get_filename_from_1password(item_id) or DEFAULT_ENV_FILE_NAME
 
-    secrets = _get_file_contents(env_file_name)
+    secrets = _get_file_contents(env_file_name, raise_if_not_found=True)
```

Comment:
When pushing secrets it does make sense for the file to exist

Reply:
the code is doing that, just added `raise_if_not_found=True` to be more explicit, the behavior is the same